### PR TITLE
Reference to SMP community group improved

### LINKED ISSUE #221: [Site] Add reference to SMP Community Group
**Current State of the service-mesh-performance site**
SMP has a new [Community Group](https://community.cncf.io/service-mesh-performance-community/) in Cloud Native Community Groups.

**Desired State of the service-mesh-performance site**
A reference to join this group should be added to the SMP website.

Community Group URL: https://community.cncf.io/service-mesh-performance-community/

### PR DESCRIPTION
**Pull request by soham4abc**
**Description**

This PR fixes #221

**Notes for Reviewers**
The SMP community grp reference centered and meeting minutes icon improved

@navendu-pottekkat  and @leecalcote  Please do have a look!

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/_includes/engage.html [hunk 1]
```diff
@@ -94,30 +94,30 @@ <h4 class="section-subheading">{{ site.data.sitetext.engage.subtext }}</h4>
...
-          <!-- Meeting Minutes -->
+          <!-- CNCF Community -->
           <div class="col-lg-4">
             <div class="card-web card hover-up hover-line"> 
               <div class="card-body">
-                <a href="https://bit.ly/3B7uIy9"><h4 class="card-title"><i class="far fa-file cmu"></i> Meeting Minutes</h4>
+                <a href="https://community.cncf.io/service-mesh-performance-community/"><h4 class="card-title"><i class="far fa-comments cmu"></i> Cloud Native Community Group</h4>
                 <p class="card-text">
-                  Have a topic to discuss? Add it to the meeting minutes.
+                  Join the Service Mesh Performance Cloud Native Community Group!
```
Comment (warunicorn19) [inline docs/_includes/engage.html:103]:
```suggestion
                  Join the SMP Cloud Native Community Group!
```

**Comment by warunicorn19** [comment]
Let's change the meeting minutes logo to the one we use right now.

**Comment by soham4abc** [comment]
> 
> 
> Let's change the meeting minutes logo to the one we use right now.

Yeah sure!

**Comment by soham4abc** [comment]
@warunicorn19  Please do have a look. Icon changed as per request!

File: docs/_includes/engage.html (hunk 1 above)
Comment (warunicorn19) [reply docs/_includes/engage.html:103]: @soham4abc if you could just do this change then we are good to go.

Comment (soham4abc) [reply docs/_includes/engage.html:103]: Yeah sure ! Doing it right away

**Comment by soham4abc** [comment]
@warunicorn19  Changes done as per request!

File: docs/_includes/engage.html [hunk 2]
```diff
@@ -94,30 +94,30 @@ <h4 class="section-subheading">{{ site.data.sitetext.engage.subtext }}</h4>
...
-          <!-- CNCF Community -->
+          <!-- Meeting Minutes -->
           <div class="col-lg-4">
             <div class="card-web card hover-up hover-line"> 
               <div class="card-body">
-                <a href="https://community.cncf.io/service-mesh-performance-community/"><h4 class="card-title"><i class="far fa-comments cmu"></i> Cloud Native Community Group</h4>
+                <a href="https://bit.ly/3B7uIy9"><h4 class="card-title"><i class="far fa-file cmu"></i> Meeting Minutes</h4>
                 <p class="card-text">
-                  Join the SMP Cloud Native Community Group!
+                  Join the Service Mesh Performance Cloud Native Community Group!
```
Comment (warunicorn19) [inline docs/_includes/engage.html:115]:
```suggestion
                 Have a topic to discuss? Add it to the meeting minutes.
```

This line will be this only, sorry my bad i should've been more clear.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]

File: docs/_includes/engage.html (hunk 2 above)
Comment (warunicorn19) [reply docs/_includes/engage.html]: line 115 has to this. while line 103 is the change that i had asked for

**Review by warunicorn19 — APPROVED** [review]
LGTM🎉! Thanks @soham4abc

**Review by navendu-pottekkat — COMMENTED** [review]
Thank you @soham4abc 

Other than swapping the cards, I do not see any changes.

Good to merge if there are no changes needed // @leecalcote

**Comment by soham4abc** [comment]
@leecalcote  Please do review!

**Comment by leecalcote** [comment]
Thank you for this ping. I overlooked this one...

**Review by leecalcote — APPROVED** [review]
Thanks, @soham4abc 👍

**Pull request merged**